docs/install/command-line-parameter-examples.md

```diff
@@ -54,9 +54,16 @@ For lists of the workloads and components that you can install by using the comm
    --includeRecommended --quiet --wait
   ```
 
-  > [!NOTE]
-  > The `--wait` parameter is designed for use in a batch file. In a batch file, execution of the next command will not continue until the installation has completed. The `%ERRORLEVEL%` environment variable will contain the return value of the command, as documented in the [Use command-line parameters to install Visual Studio](use-command-line-parameters-to-install-visual-studio.md) page.
+## Using --wait
+
+* Use in batch files or scripts to wait for the Visual Studio installer to complete before the next command is executed. For batch files, an`%ERRORLEVEL%` environment variable will contain the return value of the command, as documented in the [Use command-line parameters to install Visual Studio](use-command-line-parameters-to-install-visual-studio.md) page. Some command utilities require additional parameters to wait for completion and to get the installer's return value. The following is an example of the additional parameters used with the powershell script command 'Start-Process':
```

Comment:
Please rewrite "powershell" as "PowerShell". For more information, see the [PowerShell documentation](https://docs.microsoft.com/en-us/powershell/) page.

Reply:
Review changes, per the above comment.